Marlin/Configuration_adv.h

```diff
@@ -757,22 +757,28 @@
 
 // @section fwretract
 
-// Firmware based and LCD controlled retract
-// M207 and M208 can be used to define parameters for the retraction.
-// The retraction can be called by the slicer using G10 and G11
-// until then, intended retractions can be detected by moves that only extrude and the direction.
-// the moves are than replaced by the firmware controlled ones.
-
+/**
+ * Firmware-based and LCD-controlled retract
+ *
+ * Add G10 / G11 commands for automatic firmware-based retract / recover.
+ * Use M207 and M208 to define parameters for retract / recover.
+ *
+ * With this feature enabled, all negative E moves over the MIN_RETRACT length
```

Comment:
For clarity, you might want to change this to something like "When auto retract is enabled via M209, all negative..." since the current statement implies that auto retract is enabled by default, which is not true.